website/project/views/node.py

```diff
@@ -203,16 +203,25 @@ def project_new_node(auth, node, **kwargs):
                 http.BAD_REQUEST,
                 data=dict(message_long=e.message)
             )
-
+        redirect_url = new_component.url
         message = (
             'Your component was created successfully. You can keep working on the component page below, '
             'or return to the <u><a href="{url}">project page</a></u>.'
         ).format(url=node.url)
+        if form.inheritContributors.data != '':
```

Comment:
If you make `inheritContributors` a Boolean field, you should be able to check `is True`, although it's probably safest to just check the truthiness.